### PR TITLE
Switch to Play 3 and Pekko, add support for Scala 3

### PR DESCRIPTION
**Pull request by mkurz**
@fdimuccio All tests pass for me, also when cross testing with Scala 3. I published locally and it works also.

Can you please merge and publish as 0.10.0? Thanks!

### CODE REVIEW

File: .github/workflows/scala.yml [hunk 1]
```diff
@@ -28,7 +28,7 @@ jobs:
         distribution: 'temurin'
         cache: 'sbt'
     - name: Run tests
-      run: sbt test
+      run: sbt +test
```
Comment (mkurz) [inline .github/workflows/scala.yml:31]: Here I made sure the project is cross tests against Scala 2.13 and Scala 3.
Please make sure you cross publish when releasing, so there are artifacts for both:
- Scala 2.13: https://repo1.maven.org/maven2/com/github/fdimuccio/play2-sockjs_2.13/
- and Scala 3: https://repo1.maven.org/maven2/com/github/fdimuccio/play2-sockjs_3/ (does not exist yet of course)

Thanks!

File: build.sbt [hunk 2]
```diff
@@ -1,10 +1,10 @@
-version := "0.9.0"
+version := "0.10.0"
 
 name := "play2-sockjs"
 organization := "com.github.fdimuccio"
 
 scalaVersion := "2.13.12"
-crossScalaVersions := Seq("2.13.12")
+crossScalaVersions := Seq("2.13.12", "3.3.1")
```
Comment (mkurz) [inline build.sbt:7]: Here I add Scala 3 support :wink:

**Comment by mkurz** [comment]
Ready to review and merge :+1:

**Comment by mkurz** [comment]
@fdimuccio Can you restart the tests? Seems they were hanging? Thanks!

**Comment by mkurz** [comment]
@fdimuccio Any chance you can look at this today, thanks!

**Comment by fdimuccio** [comment]
Sorry for the late reply, It looks good, tests too. Thank you again.

**Comment by mkurz** [comment]
Great, thanks! Looking forward to the 0.10.0 release ;)

**Comment by mkurz** [comment]
Great! It's here:
- https://repo1.maven.org/maven2/com/github/fdimuccio/play2-sockjs_3/0.10.0/
- https://repo1.maven.org/maven2/com/github/fdimuccio/play2-sockjs_2.13/0.10.0/

**Pull request merged**